doc/source/intro.rst

```diff
@@ -2,11 +2,32 @@
 Introduction
 ------------
 
+GPflowOpt is a library for Bayesian Optimization with GPflow. It makes use of TensorFlow for computation of acquisition functions,
+to offer scalability, and avoid implementation of gradients. The package was created, and is currently maintained
+by `Joachim van der Herten <http://sumo.intec.ugent.be/jvanderherten>`_ and `Ivo Couckuyt <http://sumo.intec.ugent.be/icouckuy>`_
+
+Currently the software is pre-release and under construction, hence it lacks a lot of functionality and testing. This documentation
+might also be incomplete and under development. The project is open source: if you feel you have some relevant skills and are interested in contributing then please contact us on github.
```

Comment:
contact us on github by opening an issue or pull request.

-> add a direct link to the GPFlowOpt project.